src/compiler/checker.ts

```diff
@@ -46679,6 +46795,21 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
             case SyntaxKind.ImportKeyword:
             case SyntaxKind.NewKeyword:
                 return isMetaProperty(node.parent) ? checkMetaPropertyKeyword(node.parent).symbol : undefined;
+            case SyntaxKind.InstanceOfKeyword:

```

Comment:
'getSymbolAtLocation' is more and more of a misnomer -- this has become 'getUsefulLinkAtLocation'. Which is useful I guess.